src/Metrics.ts

```diff
@@ -317,6 +317,10 @@ class BrowserMetrics implements Metrics {
           this.clientNavigationHasOccurred = this.originalURL !== location.href
         }
 
+        if (metric.name === 'INP') {
+          this.metrics.inpel = metric.attribution?.eventTarget as string
```

Comment:
has this been tested somehow? The conversion of an eventTarget to string appears sketchy to me. Event target is usually a DOM element but how can a DOM element be converted into a string?

Reply:
Okay got it. I checked the documentation at https://github.com/GoogleChrome/web-vitals#inp-attribution and it looks like the eventTarget will be a string selector of the element. Looks good.